Doc/library/ftplib.rst

```diff
@@ -55,38 +55,56 @@ Reference
 FTP objects
 ^^^^^^^^^^^
 
+.. Use substitutions for some param docs so we don't need to repeat them
+   multiple places.
+
+.. |param_doc_user| replace::
+   The username to log in with.
+   If no *user* is specified, ``'anonymous'`` will be used.
+
+.. |param_doc_passwd| replace::
+   The password to use when logging in.
+   If not given, and if *passwd* is the empty string or ``"-"``,
+   a password will be automatically generated.
+
+.. Ideally, we'd like to use the :ref: directive, but Sphinx will not allow it.
+
+.. |param_doc_acct| replace::
+   Account information to be used for the ``ACCT`` FTP command.
+   Few systems implement this.
+   See RFC-959 for more details.
```

Comment:
```suggestion
.. |param_doc_acct| replace::
   Account information to be used for the ``ACCT`` FTP command.
   Few systems implement this.
   See `RFC-959 <https://datatracker.ietf.org/doc/html/rfc959.html>`__
   for more details.
```

This works but it's not ideal.  I poked around a bit but I couldn't find a way to get `:rfc:` to work in there.